config/database.yml

```diff
@@ -80,6 +80,8 @@ test:
 #
 production:
   <<: *default
+  host: <%= ENV['DB_HOST'] || 'localhost' %>
```

Comment:
I think this might break existing deploys, which use `DATABASE_URL` and do not have DB_HOST or DB_PORT setup.

Reply:
Are you using database url? Could you show me the format? I couldn't get it working

Reply:
My bad, I thought this project was also using DATABASE_URL, but I just saw it's not the case, this should be fine :)

Reply:
\o/